### PR TITLE
fetch shortcode analytics

### PR DESCRIPTION
**Pull request by PUDRESCU**

### CODE REVIEW

File: lib/googl.js [hunk 1]
```diff
@@ -97,6 +98,25 @@
 
         return deferred.promise;
     }
```
Comment (kaimallea) [inline lib/googl.js]: nitpick: newline pls

**Comment by kaimallea** [comment]
Thanks @PUDRESCU ! Can you update README.md as well with an example? Then I'll merge it in. :+1:

**Comment by PUDRESCU** [comment]
Ok, I've updated the README.md with examples.  Thanks!

**Comment by kaimallea** [comment]
:+1: Tyvm!

**Pull request merged**